packages/stockflux-launcher/src/App.js

```diff
@@ -1,69 +1,108 @@
-import React, { useRef, useEffect } from 'react';
 import cx from 'classnames';
+import { ScreenEdge, useDockWindow, useOptions } from 'openfin-react-hooks';
+import React, { useEffect, useState } from 'react';
 import { OpenfinApiHelpers } from 'stockflux-core';
-import { useDockWindow, ScreenEdge } from 'openfin-react-hooks';
-import {
-  FaChevronUp,
-  FaChevronLeft,
-  FaChevronRight,
-  FaRegHandRock
-} from 'react-icons/fa';
-import AppShortcuts from './app-shortcuts/AppShortcuts';
+import './App.css';
 import FreeTextSearch from './free-text-search/FreeTextSearch';
+import getUndockedPosition from './helpers/getUndockedPosition';
+import { ReactComponent as LeftIcon } from './icons/left.svg';
+import { ReactComponent as RightIcon } from './icons/right.svg';
+import { ReactComponent as TopIcon } from './icons/top.svg';
```

Comment:
I think we should all agree on how we are importing `svgs`. Does importing it as `ReactComponent` help in any way?

Reply:
The main reason it's imported this way is that I was getting a React error when I didn't import it as a ReactComponent, as mentioned in this thread as a common issue: https://github.com/facebook/create-react-app/issues/5293 
Agree it should be standardised though so this should be brought in line with the others

Reply:
Due to complexity of this issue, I have moved the icons to the components folder and will raise a separate ticket if we are keeping all the icons separate in their packages in future. 

Importing the images any other way either requires using a messy import, e.g.
`// eslint-disable-next-line import/no-webpack-loader-syntax
import LeftIcon from '-!react-svg-loader!./icons/left.svg';`

Or adding in `react-svg-loader` and rebuilding the entire package using webpack, which I feel is out of scope of this PR solely for the inclusion of the launcher icons